src/sql/parts/notebook/notebookActions.ts

```diff
@@ -23,13 +23,14 @@ import { noKernel } from 'sql/workbench/services/notebook/common/sessionManager'
 import { IConnectionDialogService } from 'sql/workbench/services/connection/common/connectionDialogService';
 import { NotebookModel } from 'sql/parts/notebook/models/notebookModel';
 
-const msgLoading = localize('loading', 'Loading kernels...');
-const kernelLabel: string = localize('Kernel', 'Kernel: ');
-const attachToLabel: string = localize('AttachTo', 'Attach to: ');
-const msgLoadingContexts = localize('loadingContexts', 'Loading contexts...');
-const msgAddNewConnection = localize('addNewConnection', 'Add new connection');
-const msgSelectConnection = localize('selectConnection', 'Select connection');
-const msgLocalHost = localize('localhost', 'localhost');
+const msgLoading = localize('loading', "Loading kernels...");
+const msgSwitching = localize('switching', "Switching kernels...");
```

Comment:
`Changing kernel` would be more standard. I've never seen Switching in UIs, lets go with the more vanilla option.